Source/Task/AsyncLib.cpp

```diff
@@ -771,8 +772,22 @@ STDAPI GetAsyncResult(
         else if (token != state->token)
         {
             // Call/Result mismatch.  This AsyncBlock was initiated by state->function
-            ASSERT(false);
+            char buf[100];
+            if (state->function != nullptr)
+            {
+                sprintf_s(
```

Comment:
sprintf_s is a Microsoft function, snprintf is the standard alternative

Reply:
Fixed.